doc/contribute/guidelines.rst

```diff
@@ -536,6 +536,11 @@ reference manuals, etc.
 Coding Style
 ============
 
+.. note::
```

Comment:
Am I lost here? I'm sure it used to be specifically mentioned that the coding style was snake case but had a grep and can't find it

Reply:
No, it doesn't say that, but it refers to the Linux coding style guide which does not say it either :) I will add a comment about it.